Mathlib/Algebra/Module/LinearMap/Basic.lean

```diff
@@ -171,11 +171,16 @@ theorem map_smul_inv {σ' : S →+* R} [RingHomInvPair σ σ'] (c : S) (x : M) :
 #align semilinear_map_class.map_smul_inv SemilinearMapClass.map_smul_inv
 
 /-- Reinterpret an element of a type of semilinear maps as a semilinear map. -/
-abbrev semilinearMap : M →ₛₗ[σ] M₃ where
+@[coe]
+def semilinearMap : M →ₛₗ[σ] M₃ where
```

Comment:
how to choose between def and abbrev?

Reply:
Here I chose for `def` because I don't want tactics to see through the `@[coe]` annotation. Generally `def` seems like a safer choice than `abbrev` because we want to be explicit about which definitions to unfold.